CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## [v3.3] - 2024-01-22
+
+### Added
```

Comment:
☕️💡 Looks good, is this the first `changelog` based release for this repo?

also, any reason why `package.json` version stays same? https://github.com/Azure/aks-set-context/blob/main/package.json#L3

Reply:
yes this is the first release with a changelog for this action, ive pushed an update to the package.json too